tests/configure/test_config.py

```diff
@@ -136,7 +136,7 @@ def inner_test_group():
     def test_command(api_client, x): # noqa
         click.echo(json.dumps(api_client.default_headers))
 
-    with mock.patch("databricks_cli.configure.provider.DatabricksConfig") as config_mock:
+    with mock.patch("databricks_cli.configure.config.get_config") as config_mock:
```

Comment:
This is necessary to fix this test.. https://travis-ci.org/databricks/databricks-cli/jobs/479983808